src/overlays/actors/ovl_Bg_Spot03_Taki/z_bg_spot03_taki.c

```diff
@@ -10,12 +10,15 @@
 
 #define THIS ((BgSpot03Taki*)thisx)
 
+void BgSpot03Taki_ApplyOpeningAlpha(BgSpot03Taki* this, s32 bufferIndex);
+
```

Comment:
We usually don't forward declare functions unless it's necessary. So in this case the function is defined before it's used so it doesn't have to be forward declared.
```suggestion
```